qcodes/instrument_drivers/basel/sp983c.py

```diff
@@ -13,12 +13,20 @@ class SP983C(Instrument):
     This driver supports both the SP 983 and SP 983C models. These differ only
     in their handling of input offset voltage. It is the responsibility of the
     user to capture the input offset, (from the voltage supply) and compensate
-    that as needed depending on the model.
+    that as needed for SP 983. For SP 983C model, 'input_offset_voltage'
+    argument can be used to set up offset.
 
     Note that, as this is a purely virtual driver, there is no support
     for the the remote control interface (SP 983a). It is the responsibility of
     the user to ensure that values set here are in accordance with the values
     set on the instrument.
+
+    Args:
+        name
+        input_offset_voltage: (Optional) A source input offset voltage
+            parameter. The range for input is -10 to 10 Volts and it is
+            user's responsibility to ensure this. This source parameter is
+            used to set offset voltage parameter of the preamp.
```

Comment:
I would add something like "The parameter should represent an voltage source that is connected to the "Offset Input volgate" connector of the SP983C